SwanCustomEnvironments/swancustomenvironments/scripts/builders/mamba.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Create the environment and install packages
+mamba create -p ${ENV_PATH} --file ${REQ_PATH} -y | tee -a ${LOG_PATH}
+
+# Activate the environment
+_log "Setting up the environment..."
+
+# Needed to use mamba activate without creating a new shell process
+HOME=/home/$USER mamba init
+ACTIVATE_MAMBA_CMD="source /home/$USER/.bashrc"
+eval "${ACTIVATE_MAMBA_CMD}"
+echo "${ACTIVATE_MAMBA_CMD}" >> /home/$USER/.bash_profile
```

Comment:
This means that, in `/home/$USER/.bash_profile`, we source both first the `.bashrc` in CERNBox (if it exists) and second the `.bashrc` in /home/$USER. I'm thinking whether it would be safer to source whatever the user has in CERNBox the last thing, to give priority to it.

Also, when we source /home/$USER/.bashrc, I noticed we also run:
```
eval "$(command conda shell.bash hook 2> /dev/null)"
```
And that activates a conda base environment, right? Is that what we want?

Reply:
1:
`.bash_profile` comes with these lines by default. You're saying the priority order should be first `/home/$USER/.bashrc` and then `/eos./../.bashrc`?

Reply:
I don't think so. I listed environment packages afterwards and they were there looking good.